assets/src/edit-story/components/header/headerLayout.js

```diff
@@ -6,24 +6,17 @@ import styled from 'styled-components';
 /**
  * Internal dependencies
  */
-import { CENTRAL_RIGHT_PADDING, INSPECTOR_WIDTH } from '../../constants';
 import Buttons from './buttons';
 import Title from './title';
 
 const Background = styled.div`
-	background-color: ${ ( { theme } ) => theme.colors.bg.v3 };
```

Comment:
I'd say that the background should be here. We sometimes have some scenarios where things might get under the header when moved and it should be clear that these do not protrude.

Reply:
Fixed.